.github/workflows/fw_icu4c_ci.yml

```diff
@@ -77,3 +111,43 @@ jobs:
           cd test/cintltst;
           echo 'Running ICU4C cintltst';
           make -j2 check;
+
+  debian-packaging:
+    needs: set-version-number
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
```

Comment:
```suggestion
```
I suspect we won't need the history here for the debian packaging either, but we can put it back in if we do.